actions/branch-name-to-hostname-label/main.test.ts

```diff
@@ -0,0 +1,36 @@
+import {branchNameToHostnameLabel} from './main'
+jest.mock('../utils')
+jest.mock('@actions/core')
+
+// reset the counter on mock fn calls after every test
+beforeEach(() => jest.clearAllMocks())
```

Comment:
We should switch to configure this globally: https://jestjs.io/docs/configuration#clearmocks-boolean

Reply:
TIL!